esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -429,10 +429,35 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
                             }
                         }
 
-                        m_secmgr.setown(SecLoader::loadSecManager("LdapSecurity", "EspHttpBinding", LINK(lscfg)));
-                        if(m_secmgr.get() == NULL)
+                        StringBuffer resourcesBaseDnBuf;
+                        authcfg->getProp("@resourcesBasedn", resourcesBaseDnBuf);
+
+                        StringBuffer workunitsBaseDnBuff;
+                        authcfg->getProp("@workunitsBasedn", workunitsBaseDnBuff);
+
+                        std::string instanceParams;
+                        instanceParams.append(resourcesBaseDnBuf.str()).append(",").append(workunitsBaseDnBuff.str());
+
                         {
-                            throw MakeStringException(-1, "error generating SecManager");
+                            static std::map<std::string, Owned<ISecManager>> ldapInstances;
```

Comment:
Now that this is defined outside a header it should be easier to declare it as either a static class member or a static global variable.